src/main/java/core/basesyntax/PasswordValidationException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public class PasswordValidationException extends RuntimeException {
```

Comment:
please see checklist: 
Should your custom exception extend RuntimeException or Exception and why? (double check task description)